doc/spec.md

```diff
@@ -1876,12 +1908,25 @@ The result of `set | set` is a new set whose elements are the
 union of the operands, preserving the order of the elements of the
 operands, left before right.
 
+The `^` operator accepts operands of either `int` or `set` type.
+For integers, it yields the bitwise XOR (exclusive OR) of its operands.
+For sets, it yields a new set containing elements of either first or second
+operand but not both (symmetric difference).
+
+The `<<` and `>>` operators require operands of `int` type both. They shift
+the first argument to the left or right by the number of bits given by the
+second operand.
```

Comment:
"It is a dynamic error if the second argument is negative. Implementations may impose a limit on the second operand of a left shift."